src/FluentValidation/TestHelper/ValidatorTestExtensions.cs

```diff
@@ -150,8 +167,20 @@ public static class ValidationTestExtension {
 			return failures.When(failure => failure.ErrorCode == expectedErrorCode, string.Format("Expected an error code of '{0}'. Actual error code was '{{Code}}'", expectedErrorCode));
 		}
 
-		public static IEnumerable<ValidationFailure> WithSeverity(this IEnumerable<ValidationFailure> failures, Severity expectedSeverity) {
-			return failures.When(failure => failure.Severity == expectedSeverity, string.Format("Expected a severity of '{0}'. Actual severity was '{{Severity}}'", expectedSeverity));
+		public static IEnumerable<ValidationFailure> WithoutSeverity(this IEnumerable<ValidationFailure> failures, Severity unexpectedSeverity) {
+			return failures.WhenAll(failure => failure.Severity != unexpectedSeverity, string.Format("Unexpected a severity of '{0}'", unexpectedSeverity));
+		}
+
+		public static IEnumerable<ValidationFailure> WithoutCustomState(this IEnumerable<ValidationFailure> failures, object unexpectedCustomState) {
+			return failures.WhenAll(failure => failure.CustomState != unexpectedCustomState, string.Format("Unexpected custom state of '{0}'", unexpectedCustomState));
+		}
+
+		public static IEnumerable<ValidationFailure> WithoutErrorMessage(this IEnumerable<ValidationFailure> failures, string unexpectedErrorMessage) {
+			return failures.WhenAll(failure => failure.ErrorMessage != unexpectedErrorMessage, string.Format("Unexpected an error message of '{0}'", unexpectedErrorMessage));
+		}
+
+		public static IEnumerable<ValidationFailure> WithoutErrorCode(this IEnumerable<ValidationFailure> failures, string unexpectedErrorCode) {
+			return failures.WhenAll(failure => failure.ErrorCode != unexpectedErrorCode, string.Format("Unexpected an error code of '{0}'", unexpectedErrorCode));

```

Comment:
Same for this one - grammatically it should be "Found an unexpected error code of..."